core/services/chainlink/config_test.go

```diff
@@ -918,34 +901,42 @@ func TestConfig_full(t *testing.T) {
 var invalidTOML string
 
 func TestConfig_Validate(t *testing.T) {
-	var invalid Config
-	d := toml.NewDecoder(strings.NewReader(invalidTOML)).DisallowUnknownFields()
-	require.NoError(t, d.Decode(&invalid))
-	if err := invalid.Validate(); assert.Error(t, err) {
-		got := err.Error()
-		exp := `3 errors:
-	1) EVM: 3 errors:
-		1) ChainID: invalid value 1: duplicate - must be unique
+	for _, tt := range []struct {
+		name string
+		toml string
+		exp  string
+	}{
+		{name: "invalid", toml: invalidTOML, exp: `4 errors:
+	1) Database: Lock: LeaseRefreshInterval (6s) must be less than or equal to half of LeaseDuration (10s)
+	2) EVM: 3 errors:
+		1) 1: ChainID: invalid value 1: duplicate - must be unique
 		2) 0: Nodes: 3 errors:
-				1) Name: invalid value foo: duplicate - must be unique
+				1) 1: Name: invalid value foo: duplicate - must be unique
 				2) 0: HTTPURL: missing: required for all nodes
 				3) 1: 2 errors:
```

Comment:
What does the `1:` and `0:` mean?

Reply:
The index of the invalid element, since these are arrays.